source/core/slang-shared-library.h

```diff
@@ -94,6 +94,13 @@ class TemporarySharedLibrary : public DefaultSharedLibrary
     String m_path;
 };
 
+class SharedLibraryUtils
```

Comment:
Perhaps these functions should be placed in 'SharedLibrary' in slang-platform.h/.cpp? It could be argued that SharedLibrary class name is not great in those files. 

It could be argued that slang-shared-library should contain all this stuff (not just the 'object' version of the interface.